core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -143,6 +143,7 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
           inProgress.put(topicAndPartition, LogCleaningPaused)
         case Some(state) =>
           state match {
+            case LogCleaningPaused => {}
```

Comment:
If you catch this here, the error message for the default case below needs to be modified since it refers to both aborted and paused states, but this handles the paused state.